cmd/api/main.go

```diff
@@ -419,3 +396,77 @@ func createChainIDStack(
 		},
 	}, nil
 }
+
+func configureTelemetry(
```

Comment:
All the wiring of the telemetry stuff happens in this function.
A new person can simply jump to this function to understand how the wiring works.
Also, the parameters clearly show configurations and dependencies needed.